posts/enterprise-lacking-cloud-skills.md

```diff
@@ -0,0 +1,63 @@
+# Enterprise lacking skills required for cloud migration
+
+> 66% of I.T. leaders say their organisations are not adequately prepared for the disruptions to come
+
+http://www.gartner.com/newsroom/id/3375817
+
+For all the excitement and talk of cloud technologies it seems the reality is a little less glamorous. With more and more executives backing the move to the cloud, more and more organisations are booting up large migration programmes. The problem is, organisations do not necessarily have the skills to migrate. 66% of I.T. leaders agree in a Gartner report that their organisations are not adequately prepared for this change.
```

Comment:
You don't need both the opening quote and the final sentence (which essentially repeats the quote). Would just link to the Gartner report in this paragraph.

Reply:
Same for the two instances below (plus in this format it'll be difficult to recreate in Sir Trev).